app/components/funding/view.rb

```diff
@@ -5,10 +5,9 @@ class View < GovukComponent::Base
     include SanitizeHelper
     include FundingHelper
 
-    attr_accessor :data_model
-
-    def initialize(data_model:)
+    def initialize(data_model:, error: false)
```

Comment:
I wonder if we can make this `error` arg a little more descriptive. It's just passed down to `MappableFieldRow` right?

Reply:
That's correct, it is just passed down. Updated globally to `has_errors`.